packages/@aws-cdk/aws-apigateway/lib/stepfunctions-api.ts

```diff
@@ -0,0 +1,128 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { RestApi, RestApiProps } from '.';
+import { RequestContext } from './integrations';
+import { StepFunctionsExecutionIntegration } from './integrations/stepfunctions';
+import { Model } from './model';
+
+/**
+ * Properties for StepFunctionsRestApi
+ *
+ */
+export interface StepFunctionsRestApiProps extends RestApiProps {
+/**
+ * The default State Machine that handles all requests from this API.
+ *
+ * This stateMachine will be used as a the default integration for all methods in
+ * this API, unless specified otherwise in `addMethod`.
+ */
+  readonly stateMachine: sfn.IStateMachine;
+
+  /**
+   * You can add requestContext (similar to input requestContext from lambda input)
+   * to the input. The 'requestContext' parameter includes account ID, user identity, etc.
+   * that can be used by customers that want to know the identity of authorized users on
+   * the state machine side. You can individually select the keys you want by setting them to true.
+   * The following code defines a REST API like above but also adds 'requestContext' to the input
+   * of the State Machine:
+   *
+   * @default - all parameters within request context will be set as false
+   */
```

Comment:
Some missing code referenced here.

Can you just copy-paste from the other doc that is well written already?